kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/usb/UsbDeviceAddedEvent.java

```diff
@@ -102,4 +102,14 @@ public String getUsbBusNumber() {
     public String getUsbDevicePath() {
         return (String) getProperty(USB_EVENT_DEVICE_PATH_PROPERTY);
     }
+
+    /**
+     * Returns the USB device type.
+     *
+     * @return
```

Comment:
I know that it has not been specified before, but I would like to document that it returns null (if we don't want to manage that differently) if the property is not available.